package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "ember-router-scroll",
-  "version": "0.5.0",
+  "version": "0.6.0",
   "description": "Scroll to top with preserved browser history scroll position",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
   "scripts": {
     "build": "ember build",
-    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "eslint": "eslint . --fix",
+    "lint:ci": "eslint .",
```

Comment:
Same with these commands. `lint:js` is added by ember-cli and will be expected to exist by most users. We should strive to keep things as close to the recommended path as possible, to make it easiest for contributors.